code/game/objects/items/devices/flashlight.dm

```diff
@@ -216,6 +216,9 @@
 	else
 		update_brightness()
 
+/obj/item/device/flashlight/flare/ground_act(var/obj/item/weapon/reagent_containers/RC)
```

Comment:
> Nonstandard formatting

No this is DEFINITELY standard formatting. BYOND explodes if you do named argument handling and some jackass decided he was too lazy to copy paste over the function signature properly.